Casks/b/bambu-connect@beta.rb

```diff
@@ -0,0 +1,23 @@
+cask "bambu-connect@beta" do
```

Comment:
```suggestion
cask "bambu-connect" do
```

We only add variants of already existing casks. If the development version is the most stable release, then we can simply use the `bambu-connect` token and transition to stable releases once they become available.